flagd/pkg/service/constants.go

```diff
@@ -0,0 +1,3 @@
+package service
+
+const FLAGD_SELECTOR_HEADER = "Flagd-Selector"
```

Comment:
This is used for the selector in OFREP and connect (gRPC over HTTP/1.1). Apparently `X-` prefixes are not to be used for custom headers anymore.